### ISSUE TITLE
No fasta files error

### ISSUE THREAD

**Issue by lpoorvin** [MaestSi/ONTrack#2]
I've just installed (I think successfully) ONTrack but can't get it to work properly.
Running,
`Rscript ONTrack.R /path/to/fasta/file`
returns
`Error: No fasta files in directory /path/to/fasta/file
Execution halted`
There is both a fasta and fastq file in the directory that I pointed to.  Am I missing something basic?
Thanks for any help you can offer!

**Comment by MaestSi** [comment]
Hi, are the fasta and the fastq files named BC\<numbers\>.fast*? ONTrack.R script looks for files that follow that naming convention. So, for example, you could rename your files BC01.fasta and BC01.fastq.
Did this solve the issue?

**Comment by lpoorvin** [comment]
Yep, that was it.  I had demuxed with qcat which names the files "barcodeXX.fastq".  Thanks!

**Comment by MaestSi** [comment]
Your are welcome!
Simone

**Comment by ieknot** [comment]
Hello, I seem to have the same problem, but simply renaming the file BC01.fastq is not working. I made a new folder for the one .fastq file I have, but it seems to not recognise it. 

I'm trying this with a file that's from one of my first ever MinION runs from 2 years ago, and I didn't use barcodes for this specific run. Is that a problem for this pipeline? So that experiment was one amplicon without barcodes, ran on one flow cell.

**Comment by MaestSi** [comment]
Hi,
the full pipeline from fast5 to consensus is not going to run for one sample only, as the current implementation doesn't allow to skip demultiplexing. I might add that feature, if you plan to need it frequently. However, the `ONTrack.R` script should work. Did you also create the corresponding fasta file with:
`seqtk seq -A BC01.fastq > BC01.fasta`?
If this is the case, what kind of error are you getting?
Could you also post the command you used?
Thanks,
Simone

**Comment by ieknot** [comment]
Thanks for your quick reply! I didn't do this indeed, but creating the fasta file helped and it ran! The command I ran was the same as in the original post: `Rscript ONTrack.R /path/to/fasta/file`. It just wasn't quite clear to me that both a fasta file and fastq file were necessary.

The `ONTrack.R` script indeed worked. I basecalled my reads with Guppy and trimmed my reads with Porechop, so I can apply a similar process for those few non-barcoded samples I have. I'm planning to mostly work with barcoded samples, so don't require the pipeline to be able to skip demultiplexing. But thanks for offering!
